src/scss/page-types/_campaign-page--fullwidth-image.scss

```diff
@@ -21,12 +21,16 @@
     }
 
     .campaign-page__hero-content-logo {
-      max-width: 43rem;
+      max-width: 100%;
+
+      @include grid-media('2xl') {
+        max-width: 43rem;
+      }
     }
 
     .campaign-page__hero-content {
       &-wrapper {
-        @include padding(3.4rem 2rem);
+        @include padding(3.4rem);
```

Comment:
Padding for the campaign hero content box was handled strangely so cleaned that up a bit as well